docs/library/ujson.rst

```diff
@@ -12,11 +12,11 @@ data format.
 Functions
 ---------
 
-.. function:: dump(obj, stream)
+.. function:: dump(obj, stream, separators=None)
```

Comment:
Since `indent` is also added that should probably be mentioned here as well?

Reply:
I have added `indent` to the docs as well and copied the comment for the CPython docs for `separators`, adapted for the not supported `indent` argument.